Tests/Functional/Controller/ImportTest.php

```diff
@@ -7,185 +7,232 @@
 
 namespace ShopwarePlugins\Connect\Tests\Functional\Controller;
 
-use ShopwarePlugins\Connect\Tests\DatabaseTestCaseTrait;
+use ShopwarePlugins\Connect\Tests\TestClient;
+use ShopwarePlugins\Connect\Tests\WebTestCaseTrait;
+use Symfony\Component\HttpFoundation\Response;
 
-class ImportTest extends \Enlight_Components_Test_Plugin_TestCase
+class ImportTest extends \PHPUnit_Framework_TestCase
 {
-    use DatabaseTestCaseTrait;
+    use WebTestCaseTrait;
 
-    public function setUp()
+    /**
+     * @param Response $response
+     * @return array
+     */
+    private function handleJsonResponse(Response $response)
```

Comment:
this could be part of the WebTestCase trait or not?
why copy it all over every testclass

Reply:
I am not sure putting about putting assert calls into a trait, because that implies that the trait resides in a subclass of PHPUnit_Framework_TestCase